### ISSUE TITLE
About the Checkpoint

### ISSUE THREAD

**Issue by 2862909626** [NVlabs/edm#14]
I want to know which checkpoint in the pretrain model is ccorresponding to the model trained by EDMloss in loss.py? In Table 1 in the paper, the EDM loss is not divided into vp and ve, but in the pretrain files I can find xxxx_vp.pkl or xxxx_ve.pkl. What's the different between xxxx_vp.pkl and xxxx_ve.pkl in pretrain files (not in baselines files)?

**Comment by Godzilla0517** [comment]
same question!
